test/integration/scheduler_perf/util.go

```diff
@@ -126,16 +126,20 @@ func mustSetupScheduler(ctx context.Context, b *testing.B, config *config.KubeSc
 	// be applied to start a scheduler, most of them are defined in `scheduler.schedulerOptions`.
 	_, informerFactory := util.StartScheduler(ctx, client, cfg, config)
 	util.StartFakePVController(ctx, client, informerFactory)
+	runGC := util.CreateGCController(ctx, tb, *cfg, informerFactory)
+	runNS := util.CreateNamespaceController(ctx, tb, *cfg, informerFactory)
```

Comment:
They are needed because tests block until all objects are truly gone before the next test starts.

Even if they are not strictly needed in performance tests (because there everything gets reset between tests, including etcd), IMHO it makes them a bit more realistic. But I can also skip them when running performance tests.

Reply:
I'm indifferent, but please update the name of the function and/or comment to reflect what's happening inside the function and why.

Reply:
I renamed "setup scheduler" to "setup cluster".